PmipRunner.cs

```diff
@@ -59,17 +59,19 @@ private bool TryGetPmipFunction(out PmipFunctionDataItem pmipFunction)
 			if (pmipFunction != null)
 				return true;
 
-			foreach (var module in Modules)
+			foreach (var module in this._frame.RuntimeInstance.GetModuleInstances())
 			{
-				var definition = $"{{,,{module}}}mono_pmip";
-
-				PmipFunctionDataItem item = null;
-				if (!EvaluateExpression(definition, r => item = new PmipFunctionDataItem {PmipFunction = definition}))
-					continue;
-
-				pmipFunction = item;
-				_stackContext.SetDataItem(DkmDataCreationDisposition.CreateAlways, item);
-				return true;
+				if (module is DkmNativeModuleInstance)
```

Comment:
Please change to:

```csharp
var nativeModule = module as DkmNativeModuleInstance;
if (nativeModule == null)
    continue;
```

To avoid two casts and reduce nesting.